boskos/Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TAG = 0.1
+
+PROJECT ?= krzyzacy-k8s-test
+ZONE ?= us-central1-f
+CLUSTER ?= boskos-test
+
+image:
+	CGO_ENABLED=0 go build -o boskos k8s.io/test-infra/boskos/
```

Comment:
hummm... I'm not exactly know why, but I need to disable CGO in order for the image to work... because it's not cross compiling? However it should be disabled by default according to https://golang.org/cmd/cgo/#hdr-Using_cgo_with_the_go_command? @spxtr might be able to give me a lecture here.